lua/luasnip/session/snippet_collection/init.lua

```diff
@@ -254,7 +254,6 @@ function M.add_snippets(snippets, opts)
 				-- set source if it was passed, and remove from snippet.
```

Comment:
I think you need to update that comment?
```suggestion
				-- set source if it was passed.
```

Reply:
Yes, TY!